cli/src/commands/split.rs

```diff
@@ -196,26 +200,37 @@ The remainder will be in the second commit.
         commit_builder.write(tx.repo_mut())?
     };
 
-    // Mark the commit being split as rewritten to the second commit. As a
-    // result, if @ points to the commit being split, it will point to the
-    // second commit after the command finishes. This also means that any
-    // bookmarks pointing to the commit being split are moved to the second
-    // commit.
+    // Mark the first commit as the rewrite of the target commit. Since the
+    // first commit inherits the change id of the target commit, this ensures
+    // bookmarks continue pointing to the same change id before and after the
+    // split.
     tx.repo_mut()
-        .set_rewritten_commit(commit.id().clone(), second_commit.id().clone());
+        .set_rewritten_commit(commit.id().clone(), first_commit.id().clone());
```

Comment:
nit: this can be removed. It's the default behavior of`.rewrite_commit()`

Reply:
Thanks, fixed!